lib/workers/helpers.rb

```diff
@@ -4,15 +4,15 @@
 module WorkerHelpers
```

Comment:
Alternatively, we could wrap this in the new namespace as well. Either works and is fine with me!